fs/ensure_link.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+import { ensureDir, ensureDirSync } from "./ensure_dir.ts";
+import { exists, existsSync } from "./exists.ts";
+import { PathType, getFileInfoType } from "./utils.ts";
+
+/**
+ * Ensures that the hard link exists.
+ * If the directory structure does not exist, it is created.
+ *
+ * @param src the source file path. directory are not allow.
```

Comment:
hard link does not support for the directory.

Reply:
Can you add a TODO and/or issue for this?

Reply:
the system not support. This is not possible.

https://askubuntu.com/questions/210741/why-are-hard-links-not-allowed-for-directories

So I think todo is meaningless

Reply:
Ah ok. I didn’t know that.

Please change “directory are not allow” to “Directory hard links are not allowed”

Reply:
Ah ok. I didn’t know that.

Please change “directory are not allow” to “Directory hard links are not allowed”